nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestPutEmail.java

```diff
@@ -137,7 +137,7 @@ public void testOutgoingMessage() throws Exception {
     public void testOutgoingMessageWithOptionalProperties() throws Exception {
         // verifies that optional attributes are set on the outgoing Message correctly
```

Comment:
I don't see the non-ascii, am I missing something?

Reply:
The test is for the non-ascii accented 'i' here in the first 'i' of "TestingNífi" on lines 140/167 as well as for the accented 'u' in "Búlk"